docket.mk

```diff
@@ -1,12 +1,27 @@
-%_import : %.csv
+%_import : %.csv sfm_pc/management/commands/country_data sfm_download.zip
 	perl -pe "s/,/ /g" $< | \
 	xargs -L1 bash -c ' \
-		echo "Importing data for country code $$3" && (\
+		echo "Loading data for country code $$3" && (\
 			python -u manage.py import_google_doc \
-				--source_doc_id $$0 \
+				--country_code $$3 \
+				--country_directory country_data/countries/$$3 \
+				--sources_path country_data/sources.csv || \
+			exit 255 \
+		)'
+
+sfm_download.zip : sfm_pc/management/commands/country_data
+	zip -r $@ $<
+
+sfm_pc/management/commands/country_data : import_docket.csv
```

Comment:
I'm not sure I understand Make enough to get what you're saying, could you show me a code example @hancush? My understanding is that is already a target. Are you suggesting to do something like

```
country_data: sfm_pc/management/commands/country_data
```

so you can just run `make country_data` instead of the entire directory path?

Reply:
Or are you suggesting to make all of the individual country files a target?

Reply:
Ah, sorry, I misread your Makefile.